data/projects.ts

```diff
@@ -39,6 +39,7 @@ import { zkitter } from "./projects/zkitter"
 import { zkml } from "./projects/zkml"
 import { Zkopru } from "./projects/zkopru"
 import { zkp2p } from "./projects/zkp2p"
+import { chiquito } from "./projects/chiquito"
```

Comment:
@leolara only one missing thing: this needs to be added to the `projects` (line 55-56) list below. 

here a detailed overview of the steps to follow https://github.com/privacy-scaling-explorations/website-v2/blob/main/data/projects/README.md